src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const ADD_OPTION = 'addProperties';
+  const REMOVE_OPTION = 'removeProperties';
+  const CLEAR_OPTION = 'clear';
+
+  for (const action of actions) {
+    switch (action.type) {
+      case ADD_OPTION:
+        for (const key in action.extraData) {
```

Comment:
Checklist: [CODE STYLE]: don't use `for in` loop for iterating over array

Reply:
It is an outdated file